petab/core.py

```diff
@@ -105,6 +126,8 @@ def get_observable_id(parameter_id: str) -> str:
     Returns:
         Observable ID
     """
+    warn("This function will be removed in future releases.",
```

Comment:
... because that is not petab's topic, right?

Reply:
This was based on the naming convention that we just kicked out. So no need to keep it, but let's phase it out slowly. I think I used it in the migration function to update models of the old format.